### ISSUE TITLE
Build problem on Windows 10

### ISSUE THREAD

**Issue by mfilippov** [dotnet/source-build#205]
Build problem on Windows 10:
```
Done building target "CopyPackage" in project "repository.proj".
Target "WriteVersions" in project "C:\Work\source-build\targets\repository.proj" (target "BuildRepositoryAndDependencies" depends on it):
Using "WriteVersionsFile" task from assembly "C:\Work\source-build\tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks/bin/Debug/netstandard1.5/Microsoft.DotNet.SourceBuild.Tasks.dll".
Task "WriteVersionsFile"
Done executing task "WriteVersionsFile".
Done building target "WriteVersions" in project "repository.proj".
Target "BuildRepositoryAndDependencies" in project "C:\Work\source-build\targets\repository.proj" (entry point):
Done building target "BuildRepositoryAndDependencies" in project "repository.proj".
Done Building Project "C:\Work\source-build\targets\repository.proj" (BuildRepositoryAndDependencies target(s)).
Done executing task "MSBuild" -- FAILED.
Done building target "BuildRepositoryReferences" in project "repository.proj" -- FAILED.
Done Building Project "C:\Work\source-build\targets\repository.proj" (BuildRepositoryAndDependencies target(s)) -- FAILED.
Done executing task "MSBuild" -- FAILED.
Done building target "Build" in project "build.proj" -- FAILED.
Done Building Project "C:\Work\source-build\build.proj" (default targets) -- FAILED.

Build FAILED.

"C:\Work\source-build\build.proj" (default target) (1) ->
"C:\Work\source-build\tools-local\init-build.proj" (default target) (2) ->
"C:\Work\source-build\targets\repository.proj" (ApplyPatches target) (3:2) ->
(ApplyPatches target) ->
  EXEC : warning : build.sh has type 100644, expected 100755 [C:\Work\source-build\targets\repository.proj]


"C:\Work\source-build\build.proj" (default target) (1) ->
"C:\Work\source-build\tools-local\init-build.proj" (default target) (2) ->
"C:\Work\source-build\targets\repository.proj" (ApplyPatches target) (3:2) ->
  EXEC : warning : build.sh has type 100644, expected 100755 [C:\Work\source-build\targets\repository.proj]


"C:\Work\source-build\build.proj" (default target) (1) ->
"C:\Work\source-build\tools-local\init-build.proj" (default target) (2) ->
"C:\Work\source-build\targets\repository.proj" (ApplyPatches target) (3:3) ->
  EXEC : warning : src/Native/build-native.sh has type 100644, expected 100755 [C:\Work\source-build\targets\repository.proj]


"C:\Work\source-build\build.proj" (default target) (1) ->
"C:\Work\source-build\tools-local\init-build.proj" (default target) (2) ->
"C:\Work\source-build\targets\repository.proj" (ApplyPatches target) (3:9) ->
  EXEC : warning : scripts/build.sh has type 100644, expected 100755 [C:\Work\source-build\targets\repository.proj]


"C:\Work\source-build\build.proj" (default target) (1) ->
"C:\Work\source-build\tools-local\init-build.proj" (default target) (2) ->
"C:\Work\source-build\targets\repository.proj" (ApplyPatches target) (3:9) ->
  EXEC : warning : scripts/build.sh has type 100644, expected 100755 [C:\Work\source-build\targets\repository.proj]


"C:\Work\source-build\build.proj" (default target) (1) ->
"C:\Work\source-build\tools-local\init-build.proj" (default target) (2) ->
"C:\Work\source-build\targets\repository.proj" (ApplyPatches target) (3:9) ->
  EXEC : warning : scripts/build.sh has type 100644, expected 100755 [C:\Work\source-build\targets\repository.proj]


"C:\Work\source-build\build.proj" (default target) (1) ->
"C:\Work\source-build\tools-local\init-build.proj" (default target) (2) ->
"C:\Work\source-build\targets\repository.proj" (ApplyPatches target) (3:9) ->
  EXEC : warning : scripts/build.sh has type 100644, expected 100755 [C:\Work\source-build\targets\repository.proj]


"C:\Work\source-build\build.proj" (default target) (1) ->
"C:\Work\source-build\tools-local\init-build.proj" (default target) (2) ->
"C:\Work\source-build\targets\repository.proj" (ApplyPatches target) (3:9) ->
  EXEC : warning : scripts/build.sh has type 100644, expected 100755 [C:\Work\source-build\targets\repository.proj]


"C:\Work\source-build\build.proj" (default target) (1) ->
"C:\Work\source-build\tools-local\init-build.proj" (default target) (2) ->
"C:\Work\source-build\targets\repository.proj" (ApplyPatches target) (3:9) ->
  EXEC : warning : scripts/build.sh has type 100644, expected 100755 [C:\Work\source-build\targets\repository.proj]


"C:\Work\source-build\build.proj" (default target) (1) ->
"C:\Work\source-build\tools-local\init-build.proj" (default target) (2) ->
"C:\Work\source-build\targets\repository.proj" (ApplyPatches target) (3:12) ->
  EXEC : warning : build.sh has type 100644, expected 100755 [C:\Work\source-build\targets\repository.proj]
  EXEC : warning : tools/Build.sh has type 100644, expected 100755 [C:\Work\source-build\targets\repository.proj]


"C:\Work\source-build\build.proj" (default target) (1) ->
"C:\Work\source-build\tools-local\init-build.proj" (default target) (2) ->
"C:\Work\source-build\targets\repository.proj" (ApplyPatches target) (3:12) ->
  EXEC : warning : build-core.sh has type 100644, expected 100755 [C:\Work\source-build\targets\repository.proj]


"C:\Work\source-build\build.proj" (default target) (1) ->
"C:\Work\source-build\tools-local\init-build.proj" (default target) (2) ->
"C:\Work\source-build\targets\repository.proj" (ApplyPatches target) (3:14) ->
  EXEC : warning : build_projects/update-dependencies/update-dependencies.sh has type 100644, expected 100755 [C:\Work\source-build\targets\repository.proj]
  EXEC : warning : run-build.sh has type 100644, expected 100755 [C:\Work\source-build\targets\repository.proj]


"C:\Work\source-build\build.proj" (default target) (1) ->
"C:\Work\source-build\tools-local\init-build.proj" (default target) (2) ->
"C:\Work\source-build\targets\repository.proj" (ApplyPatches target) (3:14) ->
  EXEC : warning : build_projects/update-dependencies/update-dependencies.sh has type 100644, expected 100755 [C:\Work\source-build\targets\repository.proj]


"C:\Work\source-build\build.proj" (default target) (1) ->
"C:\Work\source-build\tools-local\init-build.proj" (default target) (2) ->
"C:\Work\source-build\targets\repository.proj" (ApplyPatches target) (3:17) ->
  EXEC : warning : src/buildfromsource.sh has type 100644, expected 100755 [C:\Work\source-build\targets\repository.proj]


"C:\Work\source-build\build.proj" (default target) (1) ->
"C:\Work\source-build\tools-local\init-build.proj" (default target) (2) ->
"C:\Work\source-build\targets\repository.proj" (ApplyPatches target) (3:17) ->
  EXEC : warning : src/buildfromsource.sh has type 100644, expected 100755 [C:\Work\source-build\targets\repository.proj]


"C:\Work\source-build\build.proj" (default target) (1) ->
"C:\Work\source-build\targets\repository.proj" (BuildRepositoryAndDependencies target) (3:25) ->
"C:\Work\source-build\targets\repository.proj" (BuildRepositoryAndDependencies target) (3:27) ->
"C:\Work\source-build\targets\repository.proj" (BuildRepositoryAndDependencies target) (3:29) ->
(Build target) ->
  MASM : warning A4018: invalid command-line option : /Yustdafx.h [C:\Work\source-build\src\coreclr\bin\obj\Windows_NT.x64.Release\src\debug\di\cordbdi.vcxproj] [C:\Work\source-build\targets\repository.proj]
  MASM : warning A4018: invalid command-line option : /FpC:/Work/source-build/src/coreclr/bin/obj/Windows_NT.x64.Release/src/debug/di/Release/stdafx.pch [C:\Work\source-build\src\coreclr\bin\obj\Windows_NT.x64.Release\src\debug\di\cordbdi.vcxproj] [C:\Work\source-build\targets\repository.proj]
  MASM : warning A4018: invalid command-line option : /Yustdafx.h [C:\Work\source-build\src\coreclr\bin\obj\Windows_NT.x64.Release\src\debug\di\cordbdi.vcxproj] [C:\Work\source-build\targets\repository.proj]
  MASM : warning A4018: invalid command-line option : /FpC:/Work/source-build/src/coreclr/bin/obj/Windows_NT.x64.Release/src/debug/di/Release/stdafx.pch [C:\Work\source-build\src\coreclr\bin\obj\Windows_NT.x64.Release\src\debug\di\cordbdi.vcxproj] [C:\Work\source-build\targets\repository.proj]


"C:\Work\source-build\build.proj" (default target) (1) ->
"C:\Work\source-build\targets\repository.proj" (BuildRepositoryAndDependencies target) (3:25) ->
"C:\Work\source-build\targets\repository.proj" (BuildRepositoryAndDependencies target) (3:34) ->
"C:\Work\source-build\targets\repository.proj" (BuildRepositoryAndDependencies target) (3:36) ->
  RestoreCommand\RequestFactory\NoOpRestoreResult.cs(27,36): warning CS1998: This async method lacks 'await' operators and will run synchronously. Consider using the 'await' operator to await non-blocking API calls, or 'await Task.Run(...)' to do CPU-bound work on a background thread. [C:\Work\source-build\src\nuget-client\src\NuGet.Core\NuGet.Commands\NuGet.Commands.csproj] [C:\Work\source-build\targets\repository.proj]


"C:\Work\source-build\build.proj" (default target) (1) ->
"C:\Work\source-build\targets\repository.proj" (BuildRepositoryAndDependencies target) (3:25) ->
"C:\Work\source-build\targets\repository.proj" (BuildRepositoryAndDependencies target) (3:27) ->
"C:\Work\source-build\targets\repository.proj" (BuildRepositoryAndDependencies target) (3:29) ->
(Build target) ->
  C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(171,5): error MSB6006: "cmd.exe" exited with code 1. [C:\Work\source-build\src\coreclr\bin\obj\Windows_NT.x64.Release\src\dlls\mscoree\coreclr\inject_debug_resources.vcxproj] [C:\Work\source-build\targets\repository.proj]
  C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(171,5): error MSB6006: "cmd.exe" exited with code 1. [C:\Work\source-build\src\coreclr\bin\obj\Windows_NT.x64.Release\src\dlls\mscoree\coreclr\inject_debug_resources.vcxproj] [C:\Work\source-build\targets\repository.proj]
  BUILD : error : native component build failed. Refer to the build log files for details: [C:\Work\source-build\targets\repository.proj]
  C:\Work\source-build\targets\repository.proj(72,5): error MSB3073: The command "C:\Work\source-build\src/coreclr//build.cmd x64 Release skiptests -PortableBuild=false " exited with code 1.

    22 Warning(s)
    4 Error(s)

Time Elapsed 00:31:16.77
Command failed to execute:  & "$CLIPATH\dotnet" "$SDKPATH/MSBuild.dll" "$SCRIPT_ROOT/build.proj" /flp:v=detailed /clp:v=detailed $captured_args
At C:\Work\source-build\build.ps1:20 char:9
+         throw "Command failed to execute: $cmd"
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Command failed ...$captured_args :String) [], RuntimeException
    + FullyQualifiedErrorId : Command failed to execute:  & "$CLIPATH\dotnet" "$SDKPATH/MSBuild.dll" "$SCRIPT_ROOT/build.proj" /flp:v=detailed /clp:v=detailed $captured_args
```

**Comment by 4creators** [comment]
Its duplicate of  #105

**Comment by eerhardt** [comment]
This is now fixed with #213
